src/styles/custom.scss

```diff
@@ -0,0 +1,74 @@
+// (C) Copyright 2014-2016 Hewlett Packard Enterprise Development LP
+
+$brand-color: #865CD6;
+$brand-neutral-colors: (#0A64A0, #DC2878, #501EB4, #49516F);
+$brand-accent-colors: (#00CCEB, #FF7D28);
+$brand-link-color: #865CD6;
+$brand-status-colors: (
+  critical: #FF324D,
+  error: #FF324D,
+  warning: #FFD602,
+  ok: #8CC800,
+  unknown: #a8a8a8,
+  disabled: #a8a8a8
+) !default;
+$brand-grey-colors: (#000001, #333333, #3B3B3B, #434343, #666666);
+
+$button-secondary-color: nth($brand-neutral-colors, 2);
+
+$brand-font-family: 'Work Sans', Arial, sans-serif;
+$brand-large-number-font-family: 'Work Sans', Arial, sans-serif;
+$code-font-family: Consolas, Menlo, 'DejaVu Sans Mono', 'Liberation Mono', monospace;
+$text-strong-font-weight: 500;
+
+$fonts-path: "https://fonts.gstatic.com/s/worksans/v2";
+
+@font-face {
```

Comment:
Is there a way to make a standard format for all @font-face so that you don't need to repeat "font-family" and "font-style"?

Reply:
You know, I'm not certain. I try to avoid stylesheets as much as possible. This is just a copy of the vanilla stylesheet, which we can edit as necessary